watertap/unit_models/ion_exchange_0D.py

```diff
@@ -599,539 +650,668 @@ def build(self):
 
         self.Pe_p = Var(
             initialize=0.1,
-            # bounds=(0.01, 0.8),
             units=pyunits.dimensionless,
             doc="Peclet particle number",
         )
 
         self.Pe_bed = Var(
```

Comment:
```suggestion
        self.N_Pe_bed = Var(
```

Reply:
modified all dimensionless numbers to match this format